scripts/smoke-test-android.ps1

```diff
@@ -22,52 +22,14 @@ Write-Host "#  |___/_|  |_|\___/|_|\_\___| |_| |___|___/  |_|   #"
 Write-Host "#                                                   #"
 Write-Host "#####################################################"
 
-if ($IsIntegrationTest)
-{
-    $BuildDir = $(GetNewProjectBuildPath)
-    $ApkFileName = "test.apk"
-    $ProcessName = "com.DefaultCompany.$(GetNewProjectName)"
-
-    if ($Action -eq "Build")
```

Comment:
We're no longer building with this script. Building happens in https://github.com/getsentry/sentry-unity/blob/main/.github/workflows/android-smoke-test-build.yml